geoextract/__init__.py

```diff
@@ -188,6 +188,10 @@ def setup(self, pipeline):
 
     def extract(self, text):
         text = self._pad(text)
+
+        if not list(self._automaton.items()):
+            raise ValueError("You must provide locations for extraction")
+
```

Comment:
I'm fine with adding this check, but it should be done at the end of `NameExtractor.setup` instead. At that point, no further words will be added to the automaton.